scanners/whatweb/parser/parser.js

```diff
@@ -0,0 +1,99 @@
+// SPDX-FileCopyrightText: 2021 iteratec GmbH
+//
+// SPDX-License-Identifier: Apache-2.0
+
+const xml2js = require('xml2js');
+
+async function parse(fileContent) {
+  const hosts = await parseResultFile(fileContent);
+  return transformToFindings(hosts);
+}
+
+function transformToFindings(targets) {
+
+  const targetFindings = targets.map(target => {
+    let tempFinding = {
+      name: target.uri,
+      category: "URL", //TODO?
+      description: target.title,
+      location: target.ipAddress,
```

Comment:
Location needs to be a full url in the scb v3 finding model.
Does using the url from the target work? Or are there case where it would not be defined?

```suggestion
      location: target.uri,
```

Reply:
I replaced target.ipAddress with target.uri for the location parameter to match the v3 finding model. This results in the situation that both name and uri state the domain in the finding, and ipAddress is moved to attributes. But I think it's best we can do?